README.md

```diff
@@ -57,11 +57,9 @@ Current Canopy-App inputs:
     | MOL              | Monin-Obukhov Length (m)                          |
     | FRP              | Total Fire Radiative Power (MW/grid cell area)    |
 
+    **Table 2. Current User Namelist Options**
 
-Current User Namelist Options:
-
-
-    | Variable Name    | Variable Descrip. (Units)                                                          |
+    | Namelist Opt.    | Namelist Descript. and Units                                                       |
```

Comment:
Was the change to "Descript." intended? The other one has "Descrip." still. There's probably enough room to spell out "Description".

Reply:
@zmoon , Sorry, let me fix this in the next [commit.](https://github.com/noaa-oar-arl/canopy-app/pull/20/commits/e331d0e9f1a79bef12127b7384f6ad0e5d5c88a9)